src/btree/bt_delete.c

```diff
@@ -271,8 +284,8 @@ __tombstone_update_alloc(
      */
     if (page_del != NULL) {
         upd->txnid = page_del->txnid;
-        upd->start_ts = page_del->timestamp;
         upd->durable_ts = page_del->durable_timestamp;
+        upd->start_ts = page_del->timestamp;
```

Comment:
What's the purpose of this change?

Reply:
I think just arbitrary alphabetical sorting.

Reply:
No, it's initializing the structure's fields in the same order as they're declared IIRC (so it's easy to check if something is missing).